src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerUpdateTranscriptionApprovedIntTest.java

```diff
@@ -241,4 +241,32 @@ void updateTranscriptionShouldReturnTranscriptionWorkflowActionInvalidError() th
         verifyNoInteractions(mockAuditApi);
     }
 
+
+    @Test
+    void updateTranscriptionApprovedWithRequestorSameAsApprover() throws Exception {
```

Comment:
maybe give the new test naming conventions a try?

Reply:
ACCEPT.